### ISSUE TITLE
list-fonts-json, command failed: version 42.2

### ISSUE THREAD

**Issue by tdubuffet** [sedwards2009/extraterm#200]
Hi,
I have the same error on version 0.42.2.
My windows: Windows 10, version 1903

![image](https://user-images.githubusercontent.com/4500077/59767324-5d594700-92a2-11e9-8ca7-a371a54c899e.png)

Thx

**Comment by tdubuffet** [comment]
Resolved ! 
Missing dll msvcp140.dll, i update visual c++

**Comment by sedwards2009** [comment]
Thanks for the detective work. It looks like I might have to also ship that DLL.

**Comment by majamee** [comment]
@sedwards2009 Unfortunatelly I get in this version on Mac basically the same error, but a Windows DLL won't probably fix that. Please find here the error message:
```Uncaught Exception:
Error: Command failed: /Applications/extraterm.app/Contents/Resources/app/extraterm/resources/list-fonts-json-binary/darwin-x64/list-fonts-json
    at checkExecSyncError (child_process.js:621:11)
    at Object.execFileSync (child_process.js:639:13)
    at Object.module.(anonymous function) [as execFileSync] (ELECTRON_ASAR.js:160:31)
    at Object.getAvailableFontsSync (/Applications/extraterm.app/Contents/Resources/app/extraterm/src/main_process/FontList.js:16:40)
    at getFonts (/Applications/extraterm.app/Contents/Resources/app/extraterm/src/main_process/Main.js:484:42)
    at main (/Applications/extraterm.app/Contents/Resources/app/extraterm/src/main_process/Main.js:94:111)
    at Object.<anonymous> (/Applications/extraterm.app/Contents/Resources/app/extraterm/src/main_process/Main.js:939:1)
    at Object.<anonymous> (/Applications/extraterm.app/Contents/Resources/app/extraterm/src/main_process/Main.js:941:3)
    at Module._compile (internal/modules/cjs/loader.js:693:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)```

**Comment by sedwards2009** [comment]
@majamee  Can you run /Applications/extraterm.app/Contents/Resources/app/extraterm/resources/list-fonts-json-binary/darwin-x64/list-fonts-json directly from a terminal. Does it output a heap of stuff about fonts?

**Comment by majamee** [comment]
@sedwards2009 It lists all fonts and at the end throws an error message. Please find here the output:
```
(...)
{
  "path": "/private/tmp/.mt1134493527/.a/.f/88.ttf",
  "postscriptName": "NeuePlak-NarrowRegular",
  "family": "Neue Plak",
  "style": "Narrow Regular",
  "weight": 400,
  "width": 3,
  "italic": false,
  "oblique": false,
  "monospace": false
}
,
{
  "path": "/Users/martinba/Library/Fonts/88ddd511-5a04-46cc-b4c3-afa01c0b7d23.ttf",
  "postscriptName": "UniversNextW04-530Medium",
Segmentation fault: 11
```

**Comment by sedwards2009** [comment]
@majamee  Can you go to this page and download the file from the middle of the page around  "macOS build output", and then run it and see if that works better. It is just a new version of `list-fonts-json` which might fix that crash.

Page- > https://dev.azure.com/simon0627/list-fonts-json/_build/results?buildId=137&view=results

**Comment by majamee** [comment]
@sedwards2009 it is working flawlessly :) thank you
P.S.: Since using your terminal on manjaro, I am looking forward to the release where I can run it on mac too 👍

**Comment by sedwards2009** [comment]
@majamee  Thanks for helping hunt that one down. There will most likely be a bug fix release in the next week or two.

**Comment by sedwards2009** [comment]
Fix is released in 0.44.0.
